test/parser.spec.js

```diff
@@ -463,6 +463,49 @@ describe('Parser', () => {
         })
       })
 
+      it('should successfully emit a computed property item with a getter', (done) => {
+        const filename = './fixtures/checkbox.vue'
+        const script = `
+          export default {
+            computed: {
+              /**
+               * ID computed prop
+               *
+               * @private
+               */
+              idGetter: {
+                get () {
```

Comment:
What if it's `set()`? 
I think dependencies prop is about getter... is not it?
If right, the parser should only parse the getter and ignore the setter function.

Reply:
I mistaken... your changes already cover this purpose.